src/main/resources/application-custom.yaml

```diff
@@ -29,7 +29,7 @@ spring:
     properties:
       hibernate.format_sql: false
       hibernate.show_sql: false
-      hibernate.dialect: ${hibernate.dialect}
+      hibernate.dialect: org.hibernate.dialect.MySQL5InnoDBDialect
```

Comment:
How can this be hard-coded if we need to support multiple dialects at least until all servers have been migrated?